reader.go

```diff
@@ -1400,14 +1410,22 @@ func (this *PdfReader) getContent(pageno int) (string, error) {
 			// Most common filter is FlateDecode which can be uncompressed with zlib
 			tmpBuffer, err := this.rebuildContentStream(contents[i])
 			if err != nil {
-				return "", errors.Wrap(err, "Failed to rebuild content stream")
+				switch err.(type) {
+				case *Ascii85DecodeError:
+					ascii85Err = err
+				default:
+					return "", errors.Wrap(err, "Failed to rebuild content stream")
```

Comment:
The way wrapping works, you can just wrap all the errors.  The client can detect if if it the wrapped error is of the specified type: https://pkg.go.dev/errors